README.md

```
@@ -50,50 +80,312 @@ A proxy server that enables Cursor IDE to use GitHub Copilot's API instead of Cu
+   npm run start:openclaw
+   ```
+
+3. Configure OpenClaw to use the proxy:
+   ```yaml
+   # openclaw.config.yaml
+   provider:
+     type: openai
+     baseUrl: http://localhost:18790/v1
+   ```
+
+3. Authenticate at `http://localhost:18790/auth.html`
```

markdownlint-cli2 (0.20.0)

[warning] Ordered list item prefix
Expected: 4; Actual: 3; Style: 1/2/3

(MD029, ol-prefix)